src/pages/EmergencyNumbers.jsx

```diff
@@ -0,0 +1,114 @@
+import { PhoneIcon } from '@chakra-ui/icons';
+import React, { useEffect, useState } from 'react';
+import { Button, Box, Stack, Text, VStack, Center, Tbody, Th, Tr, Td } from '@chakra-ui/react';
+import CommonTable from '../common/CommonTable/CommonTable';
+import { CommonTableHeader, CommonTableFooter } from '../common/CommonTable/CommonTableHeader';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'CommonTableFooter' is defined but never used.